watch/watch.1

```diff
@@ -7,41 +7,54 @@
 .SH NAME
 watch \- Keep an eye on a command output.
 .SH SYNOPSIS
-\fBwatch\fR \fB\-n\fR [\fIseconds\fR] [\fB\-bt\fR] [\fBcommand\fR [\fIargs\fR...]]
+\fBwatch\fR [\fB\-n\fR \fIseconds\fR] [\fB\-bt\fR] \fBcommand\fR [\fIargs\fR...]
 .SH DESCRIPTION
 The
 .B watch 
-utility executes a command repeatedly every "user-defined" seconds.
-This allows you to watch a program output change over the time.
+utility executes a command repeatedly every
+"user-defined" seconds. This allows you to watch
+a program output change over the time.
 .PP
 It accepts the following options:
 .TP
 .B \-n
-Specifies the interval to refresh the output. Per default, the
-interval is of 2 seconds.
+Specifies the interval to refresh the output.
+Per default, the interval is of 2 seconds.
 .TP
 .B \-b
-Beeps if the program being run returns a non-zero exit status.
+Beeps if the program being run returns a non-zero
+exit status.
 .TP
 .B \-t
-Turns off the information header, letting just the verbatim program
-output being refreshed every time on the screen \(em equivalent to a \fIksh\fR(1)
+Turns off the information header, letting just the
+verbatim program output being refreshed every time
+on the screen \(em equivalent to a \fIksh\fR(1)
 \fIfor ((;;))\fR accompanied by \fIsleep\fR(1).
 .br
-Not recommended, unless you're using \fItmux\fR(1) or another terminal multiplexer
-that shows you information that would originally be shown by that header.
+Not recommended, unless you're using \fItmux\fR(1)
+or another terminal multiplexer that shows you
+information that would originally be shown by that
+header.
 .SH EXAMPLES
-It can be very useful when backuping some disc and needing to see how many files
-were already copied.
+It can be very useful when backuping some disc and
+needing to see how many files were already copied.
```

Comment:
```suggestion
It can be very useful when backing up a disk and
you need to see how many files were already copied.
```
This is (from what I can tell) a more correct way to say this, though still kinda informal.

Reply:
Sure, I will commit those too.
Just a small curiosity: in Brazilian Portuguese, we have borrowed some computing terms directly from English instead of trying to translate those, which is done by European Portuguese and even French (like byte is not "byte", but "octet"), and "backup" is a single, agglutinated term and we often use if as a verb --- like "Eu fiz um backup daquele disco rígido" or "Faz um backup d'O Paxá, aquele filme com o Jean Moulin, deixei o DVD na sua mesa" --- so, without thinking much, I've used "backup" as a single word instead of thinking on "back" as a verb and "up" as... I do not know, I'm not good with languages in general.